<launch>
    <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch" />
    <include file="$(find explore_lite)/launch/explore.launch" />
    <include file="$(find turtlebot3_navigation)/launch/move_base.launch" />



    #<node pkg="mr" type="return_home.py" name="return_home" output="screen">
    </node>



    #<node pkg="image_transport" type="republish" name="image_republisher"
    args="compressed in:=raspicam_node/image raw out:=raspicam_node/image"/>

    <group ns="raspicam_node">
    	#<node pkg="image_proc" type="image_proc" name="image_proc">
    		<remap from="image_raw"		to="/image"/>
    		<param name="approximate_s"		type="bool"		value="true"/>
    		<param name="queue_size"		type="int"		value="20"/>
     	</node>
    </group>
    
	<node pkg="tf" type="static_transform_publisher" name="camera_rgb_optical_frame_to_cam" 
    args="0 0 0 0 0 0 camera_rgb_optical_frame camera 10" />

	<node pkg="ar_track_alvar" type="individualMarkersNoKinect" name="ar_track_alvar" respawn="false" output="screen">
		<param name="marker_size"           type="double" value="5.0" />
		<param name="max_new_marker_error"  type="double" value="0.08" />
		<param name="max_track_error"       type="double" value="0.2" />
		<param name="output_frame"          type="string" value="map" />
		<remap from="camera_image"  to="/raspicam_node/image" />
		<remap from="camera_info"   to="/raspicam_node/camera_info" />
	</node>
    
</launch>
